### ISSUE TITLE
JSON.parse: unexpected character at line 1 column 1 of the JSON data

### ISSUE THREAD

**Issue by chrismcgavic** [zowe/community#1717]
**Describe the bug**
I use https://myhosthidden:7556/ZLUX/plugins/org.zowe.zlux.bootstrap/web/ to login to desktop successfully.  When I click on the JES explorer, MVS explorer or USS explorer I'm presented with a logon screen.  I use the same credentials and get an error: JSON.parse: unexpected character at line 1 column 1 of the JSON data.

When I use developer tools to see the brower details I see this:  ERROR Error: Uncaught (in promise): Error: 404 ZLUX: unknown resource requested

Oddly enough if I use port 7554 https://myhosthidden:7554/zlux/ui/v1/ZLUX/plugins/org.zowe.zlux.bootstrap/web/ I get the desktop and click on the JES Explorer icon and get in just fine, but the MVS and USS apps aren't available

**Steps to Reproduce**

1.
2.

**Expected behavior**
I'd expect to be sent straight to the USS, MVS or JES explorer with no error or secondary signon

**Screenshots**

**Logs**
If applicable, add server logs collected at the time of your problem.

**Details**

z/OS 2.3
Zowe version: v2.3.0            
build and hash: v2.x/master#2035

**Web Browser Details (if the bug relates to Zowe Desktop usage):**
Microsoft Edge and Firefox

**Additional context**
The below is at the bottom of the link below and I have no idea where to put the allowBasicAuthLookup="false". Wondering if this would fix the issue.

https://docs.zowe.org/stable/user-guide/systemrequirements-zosmf/#configuration-of-zosmf-to-properly-work-with-api-ml

Configuration of z/OSMF to properly work with API ML

There is an issue observed in z/OSMF which leads to a stuck JSON web token(JWT). It manifests as the endpoint /zosmf/services/authenticate issuing a JWT with success RC that is not valid for API calls, resulting in 401 response status code. This is a persistent condition. To get the token unstuck, perform a logout with the LTPA token from the login request. This causes logins to start serving unique JWTs again. Until this issue is properly fixed in z/OSMF, we propose a possible temporary workaround. Update z/OSMF configuration with allowBasicAuthLookup="false". After applying this change, each authentication call results in generating a new JWT.

**Comment by chrismcgavic** [comment]
This is working now.  I had forgotten that I was using 'saf' vs 'zosmf' and changed back to 'zosmf' as apiml:security:auth:provider:.  A colleague also clued me into a server_override.xml that has the JWT code in it. The link is https://www.ibm.com/docs/en/zos/2.4.0?topic=configurations-enabling-json-web-token-support where they found it in z/OS 2.4.  I'm running z/OS 2.3 but had seen verbiage in the 2.3.0 zowe doc about making sure you had certain ptf's on for JWT support and they are on at my level of z/OS 2.3.

**Comment by chrismcgavic** [comment]
I failed to mention that I have it working using port 7554 quoted in my original message.  I just compared the z/OS 2.3 vs 2.4 zOSMF config manuals and its obvious to me that z/OS 2.3 has a more robust config guide.  If you're z/OS 2.3 you're fighting an uphill battle as I just did. Thanks to Jim and Hermann on the Teams Zowe channel for nudging me in the right direction. https://teams.microsoft.com/l/channel/19%3a2fc62a404e7d4d9091c6d46d1283df6b%40thread.tacv2/Zowe?groupId=27deb7ee-73e9-4fbc-baad-fbafdc4c361e&tenantId=f260df36-bc43-424c-8f44-c85226657b01

It's going to take a village of engineers to get zowe and zosmf working together.
